src/nvim/os/unix_defs.h

```diff
@@ -1,9 +1,10 @@
 #ifndef NVIM_OS_UNIX_DEFS_H
 #define NVIM_OS_UNIX_DEFS_H
 
-// Windows doesn't have unistd.h, so we include it here to avoid numerous
-// instances of `#ifdef WIN32'.
+// Windows doesn't have the following headers,
+// so we include them here to avoid numerous instances of `#ifdef WIN32'.
 #include <unistd.h>
+#include <sys/param.h>
```

Comment:
The comment is weird, is it still valid? `unix_defs.h` is only included by `os_defs.h`, specifically when `WIN32` is _not_ defined.